<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseInfoDialog</class>
 <widget class="QDialog" name="CourseInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="yspcclient.qrc">
    <normaloff>:/res/login/logo.png</normaloff>:/res/login/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="WidgetTop" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>77</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>77</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>240</width>
        <height>77</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/res/teacher/pcindex_head.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="PButtonClose">
      <property name="geometry">
       <rect>
        <x>995</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: bold 20px &quot;宋体&quot;;
border:none;
}</string>
      </property>
      <property name="text">
       <string>×</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="WidgetBottom" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(222, 222, 222);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>100</number>
      </property>
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>25</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget
{
background-color: rgb(255, 255, 255);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>25</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>25</number>
         </property>
         <item>
          <widget class="MyCourseWidget" name="MCWidgetCourse" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>120</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>120</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget
{
background-color: rgb(255, 255, 255);
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>25</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>25</number>
         </property>
         <property name="horizontalSpacing">
          <number>25</number>
         </property>
         <property name="verticalSpacing">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="PButtonCourse">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	background-color: transparent;
	border:none;
	font-size:15px;
	color: rgb(111, 111, 111);
}
QPushButton:checked
{
	border-left:0px;
	border-right:0px;
	border-top:0px;
	border-bottom:5px solid rgb(233,108,83);
	font-size:30px;
	color: rgb(0, 0, 0);
}
</string>
           </property>
           <property name="text">
            <string>课程</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="PButtonAbout">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	background-color: transparent;
	border:none;
	font-size:15px;
	color: rgb(111, 111, 111);
}
QPushButton:checked
{
	border-left:0px;
	border-right:0px;
	border-top:0px;
	border-bottom:5px solid rgb(233,108,83);
	font-size:30px;
	color: rgb(0, 0, 0);
}
</string>
           </property>
           <property name="text">
            <string>关于</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="PButtonMessage">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	background-color: transparent;
	border:none;
	font-size:15px;
	color: rgb(111, 111, 111);
}
QPushButton:checked
{
	border-left:0px;
	border-right:0px;
	border-top:0px;
	border-bottom:5px solid rgb(233,108,83);
	font-size:30px;
	color: rgb(0, 0, 0);
}
</string>
           </property>
           <property name="text">
            <string>留言</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>666</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="styleSheet">
            <string notr="true">QWidget#stackedWidget
{
border:none;
border-top:1px solid rgb(200,200,200);
}</string>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="PageCourse">
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QScrollArea" name="scrollArea">
               <property name="styleSheet">
                <string notr="true">QScrollArea
{
	border:none;
}
QScrollBar::vertical
{
	width: 10px;
	background:rgb(255,255,255);
	margin:0px 0px 0px 0px;
}
QScrollBar::handle:vertical
{
	background: rgb(200, 200, 200);
	border-radius:5px;
}
QScrollBar::add-line:vertical
{
border:transparent;
}
QScrollBar::sub-line:vertical
{
border:transparent;
}</string>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="SAWContentsCourse">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>774</width>
                  <height>335</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="VLayoutCourse">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="PageAbout">
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>25</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QTextEdit" name="TEditAbout">
               <property name="styleSheet">
                <string notr="true">QTextEdit
{
	border:none;
}
QScrollBar::vertical
{
	width: 10px;
	background:rgb(255,255,255);
	margin:0px 0px 0px 0px;
}
QScrollBar::handle:vertical
{
	background: rgb(200, 200, 200);
	border-radius:5px;
}
</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="PageMessage">
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="horizontalSpacing">
              <number>0</number>
             </property>
             <property name="verticalSpacing">
              <number>10</number>
             </property>
             <item row="0" column="0" colspan="3">
              <widget class="QTextEdit" name="TEditMessage">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>60</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border:none;
background-color: rgb(222, 222, 222);</string>
               </property>
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="placeholderText">
                <string>这里写下你想说的……</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="LabelMessageCount">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font-size:20px;
color: rgb(44, 44, 44);</string>
               </property>
               <property name="text">
                <string>留言(0)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>581</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="PButtonPublish">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
color: rgb(255, 255, 255);
background-color: rgb(255, 85, 0);
border-radius:15px;
font-size:18px;
}
QPushButton::disabled
{
	background-color: rgb(139, 139, 139);
}
</string>
               </property>
               <property name="text">
                <string>发表留言</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="3">
              <widget class="QScrollArea" name="scrollArea_2">
               <property name="styleSheet">
                <string notr="true">QScrollArea
{
	border:none;
}
QScrollBar::vertical
{
	width: 10px;
	background:rgb(255,255,255);
	margin:0px 0px 0px 0px;
}
QScrollBar::handle:vertical
{
	background: rgb(200, 200, 200);
	border-radius:5px;
}
QScrollBar::add-line:vertical
{
border:transparent;
}
QScrollBar::sub-line:vertical
{
border:transparent;
}</string>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="SAWContentsMessage">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>774</width>
                  <height>205</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="VLayoutMessage">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCourseWidget</class>
   <extends>QWidget</extends>
   <header>mycoursewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="yspcclient.qrc"/>
 </resources>
 <connections/>
</ui>
